services/harmony/test/helpers/ogc-api-edr.ts

```diff
@@ -94,32 +95,27 @@ export function edrRequest(
 
 /**
  * Performs getDataForCube request on the given collection with the given params
- * using a multipart/form-data POST
  *
  * @param app - The express application (typically this.frontend)
  * @param version - The OGC version
  * @param collection - The CMR Collection ID to perform a service on
- * @param form - The form parameters to pass to the request
- * @param queryString - The query string parameters to pass to the request
+ * @param form - The JSON to pass to the request
  * @returns An 'awaitable' object that resolves to a Response
  */
 export function postEdrRequest(
   app: Express.Application, version: string, collection: string, form: object, queryString = '',
 ): Test {
   let urlPathAndParam = `/ogc-api-edr/${version}/collections/${collection}/cube`;
   if (queryString) urlPathAndParam += `?${queryString}`;
-  let req = request(app)
-    .post(urlPathAndParam);
-
-  Object.keys(form).forEach((key) => {
-    if (key === 'shapefile') {
-      req = req.attach(key, form[key].path, { contentType: form[key].mimetype, filename: 'foobar' });
-
-      // req = req.attach(key, form[key].path, 'foobar');
-    } else {
-      req = req.field(key, form[key]);
-    }
-  });
+  // POST parameter-name is of type array, not string
+  form['parameter-name'] = form['parameter-name'].split(',');
+  form['granuleId'] = form['granuleId'].split(',');
```

Comment:
I made the choice of using camelCase for consistency between existing Harmony APIs (including OGC Coverages) and OGC EDR.

Reply:
For any parameters defined in EDR specification, it will be in dash-case. This is only for Harmony specific parameters which we have used camelCase. Not sure if ArcGIS type tool would require any parameters be in dash-case. For Harmony specific parameters, we have precedence of using camelCase regardless of specification preference, e.g. for WMS, the specification uses underscore in parameters, but we use camelCase for Harmony specific ones.

Reply:
Makes sense, thanks.